src/items/feats/polymorphic_titan.json

```diff
@@ -0,0 +1,184 @@
+{
+  "_id": "ywBowbHs7UabynwO",
+  "name": "Polymorphic Titan",
+  "type": "feat",
+  "_stats": {
+    "coreVersion": "10.288",
+    "createdTime": 1668818697460,
+    "lastModifiedBy": "SPRszH5sDdJlGxJn",
+    "modifiedTime": 1668892771330,
+    "systemId": "sfrpg",
+    "systemVersion": "0.21.0"
+  },
+  "img": "icons/svg/mystery-man.svg",
+  "system": {
+    "type": "",
+    "ability": null,
+    "abilityMods": {
+      "parts": []
+    },
+    "actionTarget": "",
+    "actionType": "",
+    "activation": {
+      "type": "",
+      "condition": "",
+      "cost": 0
```

Comment:
This could just be made an active feat, which would allow the modifiers to dynamically turn on or off, removing the need for the note too.